lib/classes/PluginManager.js

```diff
@@ -172,7 +172,7 @@ class PluginManager {
       const updates = updateNotifier({ pkg: sfePkgJson, interval: 1 });
       if (updates.update) {
         this.serverless.cli.log(
-          'An updated version of Serverless Enterprise is available. ' +
+          'An updated version of the Serverless Dashboard is available. ' +
```

Comment:
While this might be nitpicking I think that this message sounds a little bit strange.

Our Dashboard is a SaaS offering, so why do I need to update the Serverless Framework when the hosted Dashboard was updated? I'd rephrase it to state that the underlying package which interacts with the Dashboard needs to be updated.

I know that this might be too much, but when reading it it just doesn't feel right.